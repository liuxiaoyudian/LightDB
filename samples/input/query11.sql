SELECT Sailors.B, Sailors.C, Sailors.A FROM Sailors;