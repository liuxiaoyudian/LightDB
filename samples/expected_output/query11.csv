200,50,1
200,200,2
100,105,3
100,50,4
100,500,5
300,400,6